registrations/models.py

```diff
@@ -236,6 +239,59 @@ def is_user_editable_resources(self):
         )
 
 
+class RegistrationUser(models.Model):
+    email = models.EmailField(verbose_name=_("E-mail"), null=True, blank=True)
```

Comment:
Since this is the only new information the model holds, I would think that removing `null=True, blank=True` would probably make sense.

Reply:
Very good point: Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/631/commits/a82f25d59e6988d2771614afe85cd35f7aa5e045